docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+
+This stream has an embedded `grpc.ClientStream` and the following interface:
+
+```
+type <ServiceName>_FooClient interface {
+	Recv() (*MsgB, error)
+	grpc.ClientStream
+}
+```
+
+The stream begins when the client calls the `Foo` method on the stub. 
+The client can then repeatedly call the `Recv` method on the returned `<ServiceName>_FooClient` return value in order to read the server-to-client response stream. 
+This `Recv` method returns `(nil, io.EOF)` once the server-to-client stream has been completely read through.
+
+#### Client-Streaming methods
+These methods have the following signature on the generated client stub:
+`Foo(ctx context.Context, opts ...grpc.CallOption) (<ServiceName>_FooClient, error)`.
+
+In this context, `<ServiceName>_FooClient` represents the client-to-server `stream` of `MsgA` messages.
+
+`<ServiceName>_FooClient` has an embedded `grpc.ClientStream` and the following interface:
+
+```
+type <ServiceName>_FooClient interface {
+	Send(*MsgA) error
+	CloseAndRecv() (*MsgA, error)
+	grpc.ClientStream
+}
+```
+
+The stream begins when the client calls the `Foo` method on the stub. 
+The client can then repeatedly call the `Send` method on the returned `<ServiceName>_FooClient` stream in order to send the client-to-server message stream.
+The `CloseAndRecv` method on this stream must be called once and only once, in order to both close the client-to-server stream
+and receive the single response message from the server.
+
+#### Bidi-Streaming methods:
+These methods have the following signature on the generated client stub:
+`Foo(ctx context.Context, opts ...grpc.CallOption) (<ServiceName>_FooClient, error)`.
+
+In this context, `<ServiceName>_FooClient` represents both the client-to-server and server-to-client message streams.
+
+`<ServiceName>_FooClient` has an embedded `grpc.ClientStream` and the following interface:
+
+```
+type <ServiceName>_FooClient interface {
+	Send(*MsgA) error
+	Recv() (*MsgB, error)
+	grpc.ClientStream
+}
+```
+
+The stream begins when the client calls the `Foo` method on the stub.
+The client can then repeatedly call the `Send` method on the returned `<SericeName>_FooClient` stream in order to send the 
+client-to-server message stream. The client can also repeatedly call `Recv` on this stream in order to
+receive the full server-to-client message stream.
+
+End-of-stream for the server-to-client stream is indicated by a return value of `(nil, io.EOF)` on the `Recv` method of the stream.
+End-of-stream for the client-to-server stream can be indicated from the client by calling the `CloseSend` method on the stream.
+
+### Packages and Namespaces
+When the `prococ` compiler is invoked with `--go_out=plugins=grpc:`, the `proto package` to `Go package` "translation"
+works the same as when the `protoc-gen-go` plugin is used without the `grpc` plugin (as described in https://developers.google.com/protocol-buffers/docs/reference/go-generated).
```

Comment:
the `proto package` to Go package translation works in the same way as when using `protoc-gen-go` without the `grpc` plugin.

(you can take out the "as described in..." bit)